config/manager/manager.yaml

```diff
@@ -52,5 +52,10 @@ spec:
           requests:
             cpu: 100m
             memory: 20Mi
+        env:
```

Comment:
- when installing via bundle we'll specify the namespace in which this operator should be installed?

Reply:
yes